File: Android_Study/Example/DoitMission10_BaseApp/app/src/main/java/com/satelite54/doitmission_10_baseapp/MainActivity.java
package com.satelite54.doitmission_10_baseapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    ViewPager pager = new ViewPager(this);
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: Android_Busan_it/MyActivityIntent1/app/src/main/java/com/example/myactivityintent1/MainActivity.kt
package com.example.myactivityintent1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var btnCreateActivity : Button = findViewById<Button>(R.id.btnNewActivity)

        btnCreateActivity.setOnClickListener {
            var intent = Intent(this, SecondActivity::class.java)
            startActivity(intent);
        }
    }
}